### DISCUSSION TITLE
Azure Function hostowany poza Azurem

### DISCUSSION THREAD

**Discussion by milons** [AzureCommunityPL/group#133] [Pomoc]
Cześć. 
Czy hostowaliście kiedyś Azure Function poza Azurem? 
Mam na myśli działanie jako usługa Windows lub Linux, ewentualnie jakiś Docker? 
Może działanie na jakimś Single Board Computer jak Raspberry Pi?
Jeśli macie jakieś materiały lub doświadczenia dajcie znać, ciężko znaleźć coś w necie na ten temat.

Pytanie 2:
Czy nasłuchiwanie na zadania w kolejce na Azure Storage, Eventy Grid czy triggerowanie zadań poprzez zmiany na Blobie na Azure da się zrealizować w aplikacji konsolowej zamiast w Azure Function?

Dzięki!

**Comment by kamil-mrzyglod** [comment]
No czy ja wiem czy tak trudno - masz mnóstwo artykułów dotyczących hostowania AF z użyciem Docker. Masz też artykuł w dokumentacji związany z hostowaniem w k8s używając KEDA - https://docs.microsoft.com/en-us/azure/azure-functions/functions-kubernetes-keda
